contrib/endpoints/src/api_manager/check_auth_test.cc

```diff
@@ -594,23 +640,28 @@ TEST_F(CheckAuthTest, TestBadAudience) {
 }
 
 // Positive test: audience is service name with https prefix.
-TEST_F(CheckAuthTest, TestHttpsAudience) { TestValidToken(kTokenHttpsAud); }
+TEST_P(CheckAuthTest, TestHttpsAudience) { TestValidToken(kTokenHttpsAud); }
 
 // Positive test: audience is service name with https prefix and a trailing
 // slash.
-TEST_F(CheckAuthTest, TestHttpsSlashAudience) {
+TEST_P(CheckAuthTest, TestHttpsSlashAudience) {
   TestValidToken(kTokenHttpsSlashAud);
 }
 
 // Positive test: audience is service name with http prefix.
-TEST_F(CheckAuthTest, TestHttpAudience) { TestValidToken(kTokenHttpAud); }
+TEST_P(CheckAuthTest, TestHttpAudience) { TestValidToken(kTokenHttpAud); }
 
 // Positive test: audience is service name with http prefix and a trailing
 // slash.
-TEST_F(CheckAuthTest, TestHttpSlashAudience) {
+TEST_P(CheckAuthTest, TestHttpSlashAudience) {
   TestValidToken(kTokenHttpSlashAud);
 }
 
+INSTANTIATE_TEST_CASE_P(ServiceConfigsWithAndWithoutAudiencesCheck,
+                        CheckAuthTest,
+                        testing::Values(kServiceConfig,
```

Comment:
Actually, there are multiple tests that check for various audience fields in the token. This will just ensure that there are no regressions with this fix.